src/triage/component/architect/builders.py

```diff
@@ -333,7 +333,7 @@ def build_matrix(
             num_observations=len(output),
             lookback_duration=lookback,
             feature_start_time=matrix_metadata["feature_start_time"],
-            matrix_metadata=json.dumps(matrix_metadata, sort_keys=True, default=str),
+            matrix_metadata=matrix_metadata,
```

Comment:
As discussed, *let's see* if it's enough to serialize, properly, here.

If so, that might not be the ultimate solution; but, it would give us more flexibility.

If not, and in fact it will serialize anything you give it (including a string), then I think that's OK, and we can figure something out.